.gitignore

```diff
@@ -1,4 +1,5 @@
 /target*
 /rust.git
 /*.dot
+**/snapshots/*.snap.new
```

Comment:
Should be unnecessary because insta will automatically clean these up as you approve/reject changes? I like seeing them in `git status` so I know which ones I need to inspect.